tests/core/recipes/emt_recipes/test_emt_phonons.py

```diff
@@ -70,4 +70,22 @@ def test_phonon_flow_v4(tmp_path, monkeypatch):
     assert output["results"]["thermal_properties"]["temperatures"][-1] == 1000
     assert output["results"]["force_constants"].shape == (8, 8, 3, 3)
     assert "mesh_properties" in output["results"]
-    assert output["atoms"] != atoms
+    assert output["atoms"] == atoms
```

Comment:
Based on the test, it looks like the original `atoms` object is getting mutated somewhere? That doesn't seem ideal... 😅

Reply:
Oops, that was a me problem. Fixing...

Reply:
I think this was related to https://github.com/Quantum-Accelerators/quacc/issues/2230.